builtin/credential/cert/path_login.go

```diff
@@ -371,6 +382,33 @@ func (b *backend) checkForValidChain(chains [][]*x509.Certificate) bool {
 	return false
 }
 
+func (b *backend) checkCIDR(cert *CertEntry, req *logical.Request) error {
+
+	if len(cert.BoundCIDRs) == 0 {
+		// short-circuit the below logic
+		return nil
+	}
+
+	var valid bool
+	remoteSockAddr, err := sockaddr.NewSockAddr(req.Connection.RemoteAddr)
+	if err != nil {
+		if b.Logger().IsDebug() {
+			b.Logger().Debug("could not parse remote addr into sockaddr", "error", err, "remote_addr", req.Connection.RemoteAddr)
+		}
+		return logical.ErrPermissionDenied
+	}
+	for _, cidr := range cert.BoundCIDRs {
+		if cidr.Contains(remoteSockAddr) {
+			valid = true
```

Comment:
Rather than setting the bool, you could just `return nil` here

Reply:
:heavy_check_mark: